src/EnergyPlus/UnitarySystem.cc

```diff
@@ -10420,7 +10420,7 @@ namespace UnitarySystems {
 
         if (this->m_SuppCoilExists) {
             if (this->m_DehumidControlType_Num == DehumCtrlType::CoolReheat) {
-                if (state.dataUnitarySystems->MoistureLoad < 0.0 && this->m_HeatPump) {
+                if (state.dataUnitarySystems->MoistureLoad < 0.0 && this->m_CoolCoilExists) {
```

Comment:
The unitary system does not have to be a heat pump for zone humidity control. Instead, at least check if cooling coil exists. Maybe the cooling coil check is not necessary.

Reply:
The unitary system in this defect file has a Packaged Air Conditioner with single speed DX cooling coil, Gas Heating Coil, and an electric supplemental heating coil serving a single zone. Since the system is not a heat pump the humidity control was not activated when there is moisture load.

Reply:
I think supp heat coils were originally added for heat pumps. This seems like a reasonable change to instead check for a cooling coil.

Reply:
Thanks @Nigusse and @rraustad